recipe/meta.yaml

```diff
@@ -6,34 +6,58 @@ package:
   version: {{ version }}
 
 source:
-  fn: metis-5.1.0.tar.gz
-  md5: 5465e67079419a69e0116de24fce58fe
-  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/{{ name }}-{{ version }}.tar.gz
+  sha256: 76faebe03f6c963127dbb73c13eab58c9a3faeae48779f049066a21c087c5db2
+  patches:
+    - vc14.patch
 
 build:
-  skip: true  # [win]
   number: 3
   binary_relocation: true
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
 
 requirements:
   build:
     - cmake
     - toolchain
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
 
 test:
   commands:
-    - graphchk
-    - cmpfillin -h
-    - mpmetis -h
-    - gpmetis -h
-    - ndmetis -h
-    - gpmetis -h
-    - m2gmetis -h
-    - test -f $PREFIX/include/metis.h
+    - graphchk  # [unix]
+    - cmpfillin -h  # [unix]
+    - mpmetis -h  # [unix]
+    - gpmetis -h  # [unix]
+    - ndmetis -h  # [unix]
+    - gpmetis -h  # [unix]
+    - m2gmetis -h  # [unix]
+    - test -f $PREFIX/include/metis.h  # [unix]
+    - test -f $PREFIX/lib/libmetis.dylib  # [osx]
+    - test -f $PREFIX/lib/libmetis.so  # [linux]
```

Comment:
From the logs, it looks like the `.lib` is not generated. Usually this happens when the functions are not exported. I looked at the header file and it seems you need to define `_WINDLL`.
Try something like, `set CFLAGS="/D_WINDLL" at the top of bld.bat before invoking cmake.